samples/Console.js

```diff
@@ -0,0 +1,12 @@
+console.log("TESTING LOG");
+console.log(1);
+console.log(-1);
+console.log(123.456);
+console.log(-123.456);
+console.log({int_val: 1});
+
+console.error("TESTING ERROR");
+console.error(1);
+console.error(123.456);
+console.error(-123.456);
+console.error({int_val: 1});
```

Comment:
Yeah, I agree with @pfalcon this is more of a test than a sample. To pretend for a minute that we have a good reason for having both tests/ and sample/tests, the latter is the better place for a "developer test" like this. We don't need QA to be running it regularly, and it doesn't report PASS/FAIL like the ones in tests/ but it's still nice to share and keep around for comparison with future output perhaps.